specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-06-01-preview/keys.json

```diff
@@ -423,6 +423,10 @@
         "rotationPolicy": {
           "$ref": "#/definitions/RotationPolicy",
           "description": "Key rotation policy in response. It will be used for both output and input. Omitted if empty"
+        },
+        "releasePolicy": {
```

Comment:
If it's a goal to match the data-plane, this should be "release_policy": https://github.com/Azure/azure-rest-api-specs/blob/ee1dc806b00b73458a7d0de7b05da3c56c756cfb/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json#L1556-L1560.

Similarly, I recommend using `KeyReleasePolicy` for the model definition to be consistent. Might make troubleshooting easier, as well as dev UX.

Reply:
@heaths Made suggested changes. Please review PR again.

Reply:
Lint Checker is complaining that "release_policy" should be camelCase. Would this change be acceptable? 

<html>
<body>
<!--StartFragment-->

❌ R3016 - DefinitionsPropertiesNamesCamelCase | Property named: 'release_policy', for definition: 'KeyProperties' must follow camelCase style. Example: 'release_policy'.Location: Microsoft.KeyVault/preview/2021-06-01-preview/keys.json#L427
-- | --


<!--EndFragment-->
</body>
</html>